astroquery/cadc/cadctap/utils.py

```diff
@@ -0,0 +1,61 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
For this whole file I copied it and changed get_suitable_astropy_format() function. I couldn't figure out how to do the inheritance thing when it was just functions not classes and I don't want to change anything else in case it was needed that way for something